weather/module.py

```diff
@@ -1,23 +1,155 @@
 from typing import Optional
+import aiohttp
 
+import nextcord
 from nextcord.ext import commands
 
 import pie.database.config
-from pie import check, i18n, logger
+from pie import check, i18n, logger, utils
+from pie.i18n.database import GuildLanguage, MemberLanguage
 
 from .database import Place
 
 _ = i18n.Translator("modules/fun").translate
 guild_log = logger.Guild.logger()
 config = pie.database.config.Config.get()
 
+# number of days to get forecast for (including current day, max is 3)
```

Comment:
The maximum is set by the API, or by our implementation?

Reply:
by the API